pkg/plugin/test-keeper/plugin/file_pattern.go

```diff
@@ -0,0 +1,92 @@
+package plugin
+
+import (
+	"regexp"
+	"strings"
+)
+
+const (
+	regexDefinitionPrefix = "regex{{"
+	regexDefinitionSuffix = "}}"
+	anyPathWildcard       = "**"
+	anyNameWildcard       = "*"
+	anyNameRegex          = "[^/]*"
+	anythingRegex         = ".*"
+	twoStarsReplacement   = "<two-stars-replacement>"
+	endOfLineRegex        = "$"
+	directorySeparator    = "/"
+)
+
+// FilePattern contains regex that matches a file
+type FilePattern struct {
+	Regex string
```

Comment:
Minor thing, but if in go we call it `regexp` then maybe worth to use the same across?

Reply:
but is `regexp` more likely a name of the package/object used in the library or a convention how to name regex expressions...?